spec/lib/sarif/bundle_audit_spec.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../../spec_helper.rb'
+
+describe Sarif::BundleAuditSarif do
+  describe '#parse_issue' do
+    let(:scanner) { Salus::Scanners::BundleAudit.new(repository: repo, config: {}) }
+    before { scanner.run }
+
+    context 'scan report with logged vulnerabilities' do
+      let(:repo) { Salus::Repo.new('spec/fixtures/bundle_audit/cves_found') }
+      it 'parses information correctly' do
+        bundle_audit_sarif = Sarif::BundleAuditSarif.new(scanner.report)
+        issue = scanner.report.to_h[:info][:vulnerabilities][0]
+
+        # should Parse and fill out hash
+        expected = "Name: actionpack\nVersion: 4.1.15\nDesciption: There is a strong parameters"\
+        " bypass vector in ActionPack.\n\nVersions Affected:  rails <= 6.0.3\nNot affected:    "\
+        "   rails < 4.0.0\nFixed Versions:     rails >= 5.2.4.3, rails >= 6.0.3.1\n\nImpact\n--"\
+        "----\nIn some cases user supplied information can be inadvertently leaked from\nStrong"\
+        " Parameters.  Specifically the return value of `each`, or `each_value`,\nor `each_pair`"\
+        " will return the underlying \"untrusted\" hash of data that was\nread from the"\
+        " parameters.  Applications that use this return value may be\ninadvertently use"\
+        " untrusted user input.\n\nImpacted code will look something like this:\n\n```\ndef"\
+        " update\n  # Attacker has included the parameter: `{ is_admin: true }`\n  User.update"\
+        "(clean_up_params)\nend\n\ndef clean_up_params\n   params.each { |k, v|  SomeModel.chec"\
+        "k(v) if k == :name }\nend\n```\n\nNote the mistaken use of `each` in the"\
+        " `clean_up_params` method in the above\nexample.\n\nWorkarounds\n-----------\nDo not "\
+        "use the return values of `each`, `each_value`, or `each_pair` in your\napplication.\n"
+
+        expect(bundle_audit_sarif.parse_issue(issue)).to include(
```

Comment:
Be sure to validate all produced sarif against the json schema -that's our safety net to ensure we are producing valid SARIF.